frontend/src/App.js

```diff
@@ -10,89 +9,105 @@ function App() {
     const [isSending, setIsSending] = useState(false);
     const [address, setAddress] = useState("");
     const [selected, setSelected] = useState(null);
+    
 
     const loadChains = async () => {
-        return fetch("/api/v1/faucet/chains")
-            .then(response => response.json())
-    }
+      return fetch("/api/v1/faucet/chains").then((response) => response.json());
+    };
 
     const handleSubmit = async (event) => {
-        event.preventDefault();
-        setIsSending(true);
-        setSuccess(null);
-        setAlert(null);
-        let payload = {
-            address: address,
-            chainId: selected.chainId,
-        };
-        const response = await fetch("/api/v1/faucet/give_me", {
-            method: "POST",
-            headers: {
-                "Content-Type": "application/json"
-            },
-            body: JSON.stringify(payload)
-        });
-        let res_json = await response.json();
+      event.preventDefault();
+      setIsSending(true);
+      setSuccess(null);
+      setAlert(null);
+      let payload = {
+        address: address,
+        chainId: selected.chainId,
+      };
+      const response = await fetch("/api/v1/faucet/give_me", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(payload),
+      });
+      let res_json = await response.json();
 
-        setIsSending(false);
-        response.status === 200 ? setSuccess("your request is sent, it may takes some time to proceed") : setAlert(res_json.message);
-        response.status === 200 ? setAlert(null) : setSuccess(null);
-        console.log(response);
-    }
+      setIsSending(false);
+      response.status === 200
+        ? setSuccess("your request is sent, it may takes some time to proceed")
+        : setAlert(res_json.message);
+      response.status === 200 ? setAlert(null) : setSuccess(null);
+      console.log(response);
+    };
 
     useEffect(() => {
-        loadChains().then(res => {
-            setChains(res.chains)
-            setSelected(res.chains[0])
-        }).catch(err => {
-            console.error(err);
-            setError("server is not available");
+      loadChains()
+        .then((res) => {
+          setChains(res.chains);
+          setSelected(res.chains[0]);
         })
-    }, [])
+        .catch((err) => {
+          console.error(err);
+          setError("server is not available");
+        });
+    }, []);
 
-    if(error != null) {
-        return (
-            <div className="bg-slate-800 h-screen">
-                <div className="flex min-h-full flex-col justify-center py-12 sm:px-6 lg:px-8">
-                    <div className="sm:mx-auto sm:w-full sm:max-w-md">
-                        <AlertView message={error} type="error"/>
-                    </div>
-                </div>
-            </div>
-        )
-    }
+  //   if (error != null) {
```

Comment:
Where did our error handling go?

Reply:
![image](https://github.com/user-attachments/assets/5cde1c03-e409-4f99-babf-f0f44ee4589b)
I moved the error handling from App.js to IncoFaucet.js.

Reply:
Okay let's just delete this code rather than leaving it commented